ferrilata-devilline.IntegrationTests/Scenarios/ApiControllers/AdminController/ApiAdminAddTests.cs

```diff
@@ -0,0 +1,116 @@
+using ferrilata_devilline.IntegrationTests.Fixtures;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios
+{
+    [Collection("BaseCollection")]
+    public class ApiAdminAddTests
+    {
+        private readonly TestContext _testContext;
+        private readonly HttpContent CorrectRequestContent;
+        private readonly HttpContent IncorrectRequestContent;
+
+        public ApiAdminAddTests(TestContext testContext)
+        {
+            _testContext = testContext;
+
+            var CorrectRequestBodyObject = new { version = "2.3", name = "Badge inserter", tag = "general", levels = new List<object>() };
+            string correctRequestBody = Newtonsoft.Json.JsonConvert.SerializeObject(CorrectRequestBodyObject);
+            this.CorrectRequestContent = new StringContent(correctRequestBody, Encoding.UTF8, "application/json");
+
+            var IncorrectRequestBodyObject = new { version = "2.3", levels = new List<object>() };
+            string IncorrectRequestBody = Newtonsoft.Json.JsonConvert.SerializeObject(IncorrectRequestBodyObject);
+            this.IncorrectRequestContent = new StringContent(IncorrectRequestBody, Encoding.UTF8, "application/json");
+        }
+
+        [Fact]
+        public async Task Authorized_AndHasCorrectBody_Created()
+        {
+            var message = new HttpRequestMessage(HttpMethod.Post, "/api/admin/add");
+            message.Headers.Add("Authorization", "something");
+            message.Content = this.CorrectRequestContent;
```

Comment:
'this' keyword is not necessary

Reply:
i am totally open to eliminating every instance that is not strictly needed, but i will check with the group first the general mood (i have no idea about the this-related preferences of the other two people).